File: app/udptransport/src/main/java/org/exthmui/share/udptransport/exceptions/InvalidPacketException.java
package org.exthmui.share.udptransport.exceptions;

public class InvalidPacketException extends IllegalArgumentException {
    public InvalidPacketException() {
    }

    public InvalidPacketException(String s) {
        super(s);
    }

    public InvalidPacketException(String message, Throwable cause) {
        super(message, cause);
    }

    public InvalidPacketException(Throwable cause) {
        super(cause);
    }
}

File: app/udptransport/src/main/java/org/exthmui/share/udptransport/exceptions/InvalidPacketException.kt
package org.exthmui.share.udptransport.exceptions

class InvalidPacketException : IllegalArgumentException {
    constructor()
    constructor(s: String?) : super(s)
    constructor(message: String?, cause: Throwable?) : super(message, cause)
    constructor(cause: Throwable?) : super(cause)
}
